tensorboard/uploader/proto/write_service.proto

```diff
@@ -23,13 +27,19 @@ service TensorBoardWriterService {
   // Request that the calling user and all their data be permanently deleted.
   // Used for testing purposes.
   rpc DeleteOwnUser(DeleteOwnUserRequest) returns (DeleteOwnUserResponse) {}
+  // Request to mutate metadata associated with an experiment.
+  rpc UpdateExperimentMetadata(UpdateExperimentMetadataRequest)
```

Comment:
To echo William's comments internally, and for consistency with usual google API style, can we call this just `UpdateExperiment` (and ditto for the request and response messages) since it's acting on the `Experiment` resource directly?

Also, nit but would be nice to put it up between CreateExperiment and DeleteExperiment so that the CRUD operations fro Experiment are all adjacent to each other.

Reply:
Done, thanks for the well motivated comment!